providers/ytt/vendir/cni/_ytt_lib/addons/packages/antrea/1.7.1/bundle/config/values.yaml

```diff
@@ -46,6 +46,20 @@ antrea:
     #! also adjust MTU to accommodate for tunnel encapsulation overhead.
     defaultMTU: null
     tlsCipherSuites: TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384,TLS_RSA_WITH_AES_256_GCM_SHA384
+    enableBridgingMode: false
+    disableTXChecksumOffload: false
+    dnsServerOverride: null
+    multicastInterfaces: []
+    multicast:
+      igmpQueryInterval: "125s"
+    ipsec:
```

Comment:
Is it possible to also remove ipsec from values.yaml?

Reply:
this was synced from TCE. i  can update the antreaconfig first and try to udpate this too ( need to update tce/tf/tkg-packages, and push new versions of kapp package)